specification/maps/resource-manager/Microsoft.Maps/preview/2024-07-01-preview/maps-management.json

```diff
@@ -0,0 +1,1201 @@
+        }
+      }
+    },
+    "LinkedResources": {
+      "type": "array",
+      "maxItems": 10,
+      "items": {
+        "$ref": "#/definitions/LinkedResource"
+      },
+      "description": "The array of associated resources to the Maps account. Linked resource in the array cannot individually update, you must update all linked resources in the array together. These resources may be used on operations on the Azure Maps REST API. Access is controlled by the Maps Account Managed Identity(s) permissions to those resource(s)."
+    },
+    "LinkedResource": {
+      "type": "object",
+      "properties": {
+        "uniqueName": {
+          "type": "string",
+          "description": "A provided name which uniquely identifies the linked resource."
+        },
+        "id": {
+          "type": "string",
+          "description": "ARM resource id in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/accounts/{storageName}'."
+        }
+      },
+      "required": [
+        "uniqueName",
+        "id"
+      ],
+      "description": "Linked resource is reference to a resource deployed in an Azure subscription, add the linked resource `uniqueName` value as an optional parameter for operations on Azure Maps Geospatial REST APIs."
+    },
+    "CorsRules": {
+      "type": "object",
+      "properties": {
+        "corsRules": {
+          "type": "array",
+          "maxItems": 5,
+          "items": {
+            "description": "Specifies a CORS rule for the Maps Account.",
+            "$ref": "#/definitions/CorsRule"
+          },
+          "x-ms-identifiers": [],
+          "description": "The list of CORS rules. You can include up to five CorsRule elements in the request. "
+        }
+      },
+      "description": "Sets the CORS rules. You can include up to five CorsRule elements in the request. "
+    },
+    "CorsRule": {
+      "type": "object",
+      "properties": {
+        "allowedOrigins": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Required if CorsRule element is present. A list of origin domains that will be allowed via CORS, or \"*\" to allow all domains"
+        }
+      },
+      "required": [
+        "allowedOrigins"
+      ],
+      "description": "Specifies a CORS rule for the Map Account."
+    },
+    "Locations": {
+      "description": "The list of data processing locations for the Maps Account.",
+      "type": "array",
+      "items": {
+        "description": "Data processing location.",
+        "type": "object",
+        "properties": {
+          "locationName": {
+            "type": "string",
+            "description": "The location name."
+          }
```

Comment:
can this be an enum of location names ?

Reply:
property supports all Azure Regions. There's 69 regions right now.